compiler/computation_order/policy_chen.cc

```diff
@@ -0,0 +1,170 @@
+#include "compiler/computation_order/policy_chen.h"
```

Comment:
Ah, how about mentioning a reference to the paper (https://arxiv.org/abs/1604.06174) somewhere in this file?

Reply:
I referred the paper.